extension/app/code/community/Pagantis/Pagantis/sql/pagantis_setup/upgrade-8.3.6-8.3.8.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
the name of this file should be 8.3.7-8.3.8¿?

Reply:
I don't know but checking the filenames all start with the previous latest version:
![image](https://user-images.githubusercontent.com/1807196/82988683-d6583700-9ff9-11ea-802d-fb824aab05e9.png)